src/Agent.Listener/Configuration.Windows/NativeWindowsServiceHelper.cs

```diff
@@ -923,6 +937,40 @@ public void RevokeDirectoryPermissionForAccount(IList<string> folders)
             Trace.Info(StringUtil.Format($"Delete the group {groupName}."));
             DeleteLocalGroup(groupName);
         }
+        /// <summary>
+        /// Checks if account is managed service
+        /// </summary>
+        /// <param name="accountName">account name</param>
+        /// <returns>Returns true if account is managed service.</returns>
+        /// <exception cref="Win32Exception">Throws this exception if there is some error during check.</exception>
+        public bool IsManagedServiceAccount(String accountName)
+        {
+            bool isServiceAccount = false;
```

Comment:
After the change on line 1327, initialization is not needed.

Reply:
Fixed